### ISSUE TITLE
More options than 5?

### ISSUE THREAD

**Issue by ItchyPlant** [zjosua/anki-mc#22]
Basically the plugin is all right but what should I do if I need a "Q_6" too? 
I added it (Anki for PC allows it) but it still isn't showing up and "0 0 0 0 0 1" does not work either in the "Answers" field.

**Comment by Schlauer-Hax** [comment]
As i said in the pull request you have to:

- Add the next field to your card (Q_6.).
- Add the next line to the front of the card (<div class="hidden" id="Q_6">{{Q_6}}</div>)

This should work if you have the newest version.

**Comment by ItchyPlant** [comment]
The new field is in the position 9, the *Front Template* ends with the below lines, but the 6th option is still not showing up: 

```
...
{{#Title}}<h3 id="myH1">{{Title}}</h3>{{/Title}}
{{#Question}}<p>{{Question}}</p>{{/Question}}

<table style="boder: 1px solid black" id="qtable"></table>

<div class="hidden" id="Q_solutions">{{Answers}}</div>
<div class="hidden" id="user_answers">- - - -</div>
<div class="hidden" id="Card_Type">{{QType (0=kprim,1=mc,2=sc)}}</div>

<div class="hidden" id="Q_1">{{Q_1}}</div>
<div class="hidden" id="Q_2">{{Q_2}}</div>
<div class="hidden" id="Q_3">{{Q_3}}</div>
<div class="hidden" id="Q_4">{{Q_4}}</div>
<div class="hidden" id="Q_5">{{Q_5}}</div>
<div class="hidden" id="Q_6">{{Q_6}}</div>
```

**Comment by Schlauer-Hax** [comment]
Did you copy the new script code?

**Comment by ItchyPlant** [comment]
Now I reinstalled the add-on from this `anki-mc-v2.2.0-anki21-ankiweb.ankiaddon` file but the result was still exactly the same. 
After all, I removed the add-on, closed Anki (v2.1.35), cleaned up everything (from `C:\Users\...\AppData\Local\Anki` and `...\AppData\Roaming\Anki2`, restarted Anki, imported the add-on, and it is now working fine. 

Thank you!
